exercises/word-count/src/example.c

```diff
@@ -50,8 +50,13 @@ int word_count(char *input_text, word_count_word_t words[MAX_WORDS])
 
       // add if not yet counted else increment count.
       if (-1 == index) {
+         // reject anything that would overrun buffers
+         if (MAX_WORD_LENGTH < strlen(test_word))
+            return EXCESSIVE_LENGTH_WORD;
+         if (MAX_WORDS == unique_words)
+            return EXCESSIVE_NUMBER_OF_WORDS;
```

Comment:
leak is right here lower_case_input_text does not get freed.  I'll update tomorrow.